spec/index.html

```diff
@@ -279,61 +279,127 @@ <h2>Typographical conventions</h2>
-    <dd>An <a>object</a>
-      as specified by [[!RDF11-CONCEPTS]].</dd>
-    <dt><dfn data-cite="RDF11-CONCEPTS#dfn-rdf-triple" data-lt="rdf triple|rdf triples|triple|triples">RDF triple</dfn></dt>
-    <dd>A <a>triple</a>
-      as specified by [[!RDF11-CONCEPTS]].</dd>
-    <dt><dfn data-cite="RDF11-CONCEPTS#dfn-rdf-graph" data-lt="rdf graph|rdf graphs|graph|graphs">RDF graph</dfn></dt>
-    <dd>An <a>RDF graph</a>
-      as specified by [[!RDF11-CONCEPTS]].</dd>
-    <dt><dfn data-cite="RDF11-CONCEPTS#dfn-graph-name" data-lt="graph name|graph names">graph name</dfn></dt>
-    <dd>A <a>graph name</a>
-      as specified by [[!RDF11-CONCEPTS]].</dd>
-    <dt><dfn data-cite="RDF11-CONCEPTS#dfn-default-graph">default graph</dfn></dt>
-    <dd>The <a>default graph</a>
-      as specified by [[!RDF11-CONCEPTS]].</dd>
-    <dt><dfn data-lt="quad|quads" class="no-export">quad</dfn></dt> <!-- TODO: should be defined in N-Quads -->
-    <dd>A tuple composed of <a>subject</a>, <a>predicate</a>, <a>object</a>, and <a>graph name</a>.
-      This is a generalization of an <a>RDF triple</a> along with a <a>graph name</a>.</dd>
-    <dt><dfn data-cite="RDF11-CONCEPTS#dfn-rdf-dataset" data-lt="rdf dataset|rdf datasets|dataset|datasets">RDF dataset</dfn></dt>
-    <dd>A <a>dataset</a>
-      as specified by [[!RDF11-CONCEPTS]].
-      For the purposes of this specification, an <a>RDF dataset</a>
-      is considered to be a set of <a>quads</a></dd>
-    <dt><dfn data-cite="RDF11-CONCEPTS#dfn-blank-node">blank node</dfn></dt>
-    <dd>A <a>blank node</a>
-      as specified by [[!RDF11-CONCEPTS]]. In short, it is a node in a graph that is
-      neither an <a>IRI</a>, nor a
-      <a>literal</a>.</dd>
-    <dt><dfn data-cite="RDF11-CONCEPTS#dfn-blank-node-identifier">blank node identifier</dfn></dt>
-    <dd>A <a>blank node identifier</a>
-      as specified by [[!RDF11-CONCEPTS]]. In short, it is a <a>string</a> that begins
-      with <code>_:</code> that is used as an identifier for a
-      <a>blank node</a>. <a>Blank node identifiers</a>
-      are typically implementation-specific local identifiers; this document
-      specifies an algorithm for deterministically specifying them.</dd>
-    <dd>
-      Concrete syntaxes, like [[Turtle]] or [[N-Quads]], prepend blank node identifiers with the <code>_:</code> string
-      to differentiate them from other nodes in the graph. This affects the 
-      canonicalization algorithm, which is based on calculating a hash over the representations of <a>quads</a> in this format.
-    </dd>
-  </dl>
+  <section id="canon-terms">
+    <h3>Terms defined by this specification</h3>
+    <dl data-sort="ascending">
+      <dt><dfn data-lt="input dataset|input datasets">input dataset</dfn></dt>
+      <dd>The abstract <a>RDF dataset</a> that is provided as input to
+        the algorithm.</dd>
+      <dt><dfn>normalized dataset</dfn></dt>
+      <dd>The immutable, abstract <a>RDF dataset</a> and set of normalized
```

Comment:
```suggestion
      <dd>The immutable, abstract <a>RDF dataset</a> and a set of normalized
```

Reply:
Fixed in b6bf3e294447db5b8ba8d94c0cf23706b0997e66.